index.js

```diff
@@ -6,6 +6,14 @@ app.get('/', (req, res) => {
   res.send('Hello World!')
 })
 
+app.get('/ping', (req, res) => {
+  res.send('pon g')
+})
+
+app.get('/new', (req, res) => {
+  res.status(201).send('new')
```

Comment:
[GPT-REVIEW] The '/new' endpoint is returning a status code of 201 and a string 'new'. It would be more informative to return a JSON object with more details about the created resource.